pages/storage_and_backup/object_storage/s3_asynchronous_replication/guide.en-gb.md

```diff
@@ -0,0 +1,427 @@
+
+- **Enhanced data resiliency across regions**: Enhance your data protection strategies by replicating critical data across multiple geographical regions. This increases resiliency against data loss and ensures business continuity in the face of regional disruptions.
+
+- **Reduced latency for global access**: Positioning your data closer to your end users minimizes access latency and improves the overall user experience. Replication allows for strategic data placement in OVHcloud regions nearest to your customer base.
+
+- **Efficiency boost for computational workloads**: By bringing your data closer to your OVHcloud compute resources, replication enhances the efficiency and performance of your workloads, facilitating faster data processing and analysis.
+
+- **Compliance and regulatory fulfillment**: Many compliance frameworks mandate that data be stored at a considerable distance from the primary location or require multiple copies of critical data. Object replication simplifies the process of meeting these requirements by enabling automatic replication across vast distances and into multiple storage mediums.
+
+Implementing object replication ensures not only the safety and availability of your data but also enhances operational efficiency and compliance posture.
+
+### What is asynchronous replication?
+
+#### Basic concepts
+
+At its core, the OVHcloud Object Storage S3 Asynchronous Replication is designed to facilitate several key operations in the management and safeguarding of your data. This includes the following actions:
+
+- **Exact replica creation**
+
+![Schema 1](images/1.png)
+
+- **Replicate data within the same region**
+
+![Schema 2](images/2.png)
+
+- **Replicate data to a different region**
+  
+![Schema 3](images/3.png)
+
+- **Replicate data to two different regions**
+
+![Schema 4](images/4.png)
+
+
+### What is replicated and what is not
+
+The following table provides the default behavior of the OVHcloud Object Storage Asynchronous Replication feature:
+
+| What is replicated                                           | What is not replicated                                       |
+|--------------------------------------------------------------|--------------------------------------------------------------|
+| Objects created *after* the upload of the replication configuration | Delete marker i.e. objects deleted in the source bucket are not automatically deleted in the destination bucket |
+| Unencrypted objects                                          | Object replicas i.e. objects that are the result of a previous replication operation |
+| Objects encrypted with customer provided keys (SSE-C)        | Objects that have already been replicated to a previous destination |
```

Comment:
petite erreur dans la doc: supprimer la ligne "Objects encrypted with customer provided keys (SSE-C)" dans la colonne "What is replicated" car cette feature n'est pas encore supportée